nevergrad/benchmark/experiments.py

```diff
@@ -57,6 +57,7 @@ def mltuning(seed: tp.Optional[int] = None, overfitter: bool = False, seq: bool
               "DoubleFastGADiscreteOnePlusOne", "PSO", "BO", "MetaTuneRecentering"]
     if default_optims is not None:
         optims = default_optims
+    optims += ["Shiwa"] + ["ShiwaC" + str(i) for i in range(9)]
```

Comment:
you should use an optimizer group to create this list everywhere

Reply:
This is going to be entirely deleted actually, only one of those ShiwaC should survive.